apidef/oas/root_test.go

```diff
@@ -187,6 +187,74 @@ func Fill(t *testing.T, input interface{}, index int) {
 	}
 }
 
+// getNonEmptyFields returns non-empty fields inside a struct.
+func getNonEmptyFields(data interface{}, prefix string) (fields []string) {
```

Comment:
This would go into `internal/reflect` for a reuse + unit test needed

Reply:
This one is specific to just `TestOAS_ExtractTo_ResetAPIDefinition`. Very specific. So I believe no need to put it to a common place for reusing.